website/tutorial/basic/03-graphql-server.mdx

```diff
@@ -0,0 +1,63 @@
+---
+id: 03-graphql-server
+title: GraphQL Server
+sidebar_label: GraphQL Server
+---
+
+Now that you have a GraphQL schema, and you understand the concept of GraphQL schema and a GraphQL query, it's time to create a real GraphQL server.
+
+You are going to use the HTTP protocol to serve the GraphQL server, but note that there are other options for serving GraphQL - you can use WebSocket, SSE (Server-Sent Events) and basically any network transport protocol that you wish! (You can find a list of transport implementations at the end of this page)
+
+### Creating a GraphQL HTTP server with Yoga
+
+In this tutorial, we will be using `@graphql-yoga/node` for building our Node.js GraphQL HTTP server.
```

Comment:
Worth mentioning as a small note here that GraphQL Yoga also supports other platforms